pkgs/applications/office/appflowy/default.nix

```diff
@@ -39,7 +39,7 @@ stdenv.mkDerivation rec {
   installPhase = ''
     runHook preInstall
 
-    cd AppFlowy/
+    cd AppFlowy_x86_64-unknown-linux-gnu_ubuntu-20.04/AppFlowy/
```

Comment:
Looks like `stripRoot = false` can be removed?

Reply:
After some testing, I was able to removed it, but not the ugly `cd` path

Reply:
Did you update the hash?

Reply:
I see, I will change this

Reply:
Thanks for the reminder about changing the hash, but after changing it, I got the error message about multiple files/folders being in the archive root. To fix that, I added `stripRoot = false;` again.

Reply:
Oh, that's unfortunate. Let's keep it as is.